js/main.js

```diff
@@ -346,6 +433,24 @@ function addToSavedInData(event) {
       cardDataForWatchlist.id = $idElement.textContent;
 
       data.savedCards.unshift(cardDataForWatchlist);
+    } else if ($addToWatchlist.textContent === 'View Watchlist') {
+      switchView('watchlist');
+      $backButtonWatchlist.className = 'back-button-watchlist';
     }
   }
 }
+
+document.addEventListener('DOMContentLoaded', loadedPage);
+function loadedPage(event) {
+  $searchBarResults.value = data.searchHistory[0].keyword;
+  getMovieData(data.searchHistory[0].keyword);
+
+  switchView(data.view);
+  if (data.view === 'search-results') {
+    $searchMessage.textContent = 'Showing results for ' + '\'' + $searchBarResults.value + '\'';
```

Comment:
Can this be handled in the DOM creation process for the results page? The `load` event handler for `getMovieData`?